mptt/utils.py

```diff
@@ -6,11 +6,17 @@
 import csv
 import itertools
 import sys
+from typing import List, Optional, Tuple
```

Comment:
These seem to be unused? Typing may be a good idea but this should happen in a different PR/issue.